tool/tsh/common/tsh.go

```diff
@@ -801,6 +845,11 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	ssh.Flag("log-dir", "Directory to log separated command output, when executing on multiple nodes. If set, output from each node will also be labeled in the terminal.").StringVar(&cf.SSHLogDir)
 	ssh.Flag("no-resume", "Disable SSH connection resumption").Envar(noResumeEnvVar).BoolVar(&cf.DisableSSHResumption)
 	ssh.Flag("relogin", "Permit performing an authentication attempt on a failed command").Default("true").BoolVar(&cf.Relogin)
+	// The following flags are OpenSSH compatibility flags. They are not
+	// implemented in Kingpin but are registered here to inform developers that
+	// they are being used. Their behavior is implemented in 000000.
```

Comment:
What is the `00000` meant to reference?

Reply:
Planning on putting the commit into this once I've pushed all changes.